imports/lib/models/puzzles.js

```diff
@@ -1,14 +1,33 @@
 import { _ } from 'meteor/underscore';
+import Mustache from 'mustache';
 import { huntsMatchingCurrentUser } from '../../model-helpers.js';
 import Base from './base.js';
 import PuzzlesSchema from '../schemas/puzzles.js';
 import ActiveOperatorRole from '../active-operator-role.js';
 
-const Puzzles = new Base('puzzles', {
-  transform(doc) {
-    return _.extend({}, doc, { tags: _.uniq(doc.tags) });
-  },
-});
+// Disable escaping - we're passing this in as an attribute, not rendering it as
+// HTML, so we don't need to escape. Pretty not great that this is only
+// available as a global setting.
+Mustache.escape = s => s;
+
+const Puzzles = new class extends Base {
+  constructor() {
+    super('puzzles', {
+      transform(doc) {
+        return _.extend({}, doc, { tags: _.uniq(doc.tags) });
+      },
+    });
+  }
+
+  guessURL(hunt, puzzle) {
```

Comment:
I definitely don't love this as a place to stash this logic, but I'm not really sure where else to put it.

Reply:
I'd just as soon put it in its own file under `/imports/lib/` - maybe `guess-helpers.js` or something?  Or given that the only place it gets used is client-side, maybe even `/imports/client/guess-helpers.js`.

Reply:
Since we already have a `model-helpers.js`, I moved it there, although I think the idiomatic thing to do is probably to split each of those into their own file. /shrug problem for some other time.